modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -14,11 +14,24 @@
 namespace cv {
 namespace gapi {
 
+/// Enum class specifies output format of stereo() function
```

Comment:
M B. `The enum specified format of result that you get from @ref stereo.`

Reply:
Done

Reply:
Fixed